examples/dominantspeaker/src/helpers.js

```diff
@@ -6,13 +6,12 @@ var Video = require('twilio-video');
  * Connect to a Room with the Dominant Speaker API enabled.
  * This API is available only in Small Group or Group Rooms.
  * @param {string} token - Token for joining the Room
- * @param {string} roomName - Room name
  * @returns {CancelablePromise<Room>}
  */
-function connectToRoomWithDominantSpeaker(token, roomName) {
+function connectToRoomWithDominantSpeaker(token) {
   return Video.connect(token, {
     dominantSpeaker: true,
-    name: roomName
+    tracks: []
```

Comment:
@makarandp0 I think the code snippet that is displayed in the app should not have the empty tracks. We should pass empty tracks to the Participant that only listens to the events.

Reply:
@manjeshbhargav, This call is made by participant that only watches the tracks (the one that creates `someroom`. But I agree, that its misleading to have empty tracks in the example. It does not hurt if we created someroom is created with the tracks. I will remove the tracks option from here, and merge it in.